README.md

```diff
@@ -0,0 +1,26 @@
+# Playwright Test Project Description
+
+This project uses Playwright for automated testing of to-do-list and other apps.
+
+
+# What need to have before clone project
+
+1. Node.js >= 14.0.0
+2. npm or yarn
+3. Playwright
+
+# Getting Started
+
+1. Clone the Repository and go to your repo
+git clone https://github.com/your-username/your-repo.git
+cd your-repo
```

Comment:
Redundant steps. If someone is trying to build your project, it’s supposed to be cloned